system channel cleanup - cleanup docs (#4230)
hyperledger/fabric: docs/source/create_channel/create_channel_participation.md

# Create a channel

> To simplify the channel creation process and enhance the privacy and scalability of channels, The need for a “system channel” in order to create application channels is no longer supported. Use this tutorial to learn how to create new channels without a system channel by using the `configtxgen` tool to create a genesis block and the `osnadmin CLI` (which runs against a REST API exposed by each ordering service node) to join ordering nodes to a channel. This process allows ordering nodes to join (or leave) any number of channels as needed, similar to how peers can participate in multiple channels.

Reviewer: keep the same as before just say "it is now mandatory" instead of "it is now possible"

To simplify the channel creation process and enhance the privacy and scalability of channels, The need for a “system channel” in order to create application channels is no longer supported. Use this tutorial to learn how to create new channels without a system channel by using the `configtxgen` tool to create a genesis block and the `osnadmin CLI` (which runs against a REST API exposed by each ordering service node) to join ordering nodes to a channel. This process allows ordering nodes to join (or leave) any number of channels as needed, similar to how peers can participate in multiple channels.

**How this process differs from the legacy Fabric v2.2 process:**

>  * **System channel no longer required**: Besides the creation of the system channel representing an extra step (as compared to the new process), this system channel created an extra layer of administration that, for some use cases, provided no tangible benefit.

Reviewer: v2.2 -> v2.5 on title above

Reviewer: the system channel created

>  * **System channel no longer supported**: Besides the creation of the system channel representing an extra step (as compared to the new process), this system channel created an extra layer of administration that, for some use cases, provided no tangible benefit.

Reviewer 2: ```suggestion
 * **System channel no longer supported**: The system channel from prior releases required extra steps and administration and is no longer supported.
```

    * Orderer nodes can now track channels as a follower before being added to a channel’s consenter set, allowing them to detect this addition more quickly. Previously, this process could take several minutes, causing ordering node admins to restart their nodes so that the node could detect that it was joined more quickly.
    * If the MSP of a peer organization, as listed in the system channel, needs to be changed, the peer organization no longer needs to coordinate with an admin of the system channel to change the MSP.

> **Note:** The new mode of creating channels is incompatible with creating a channel using a system channel. If the system channel exists, the `channel join` operation is not supported. Similarly, `channel join` and `channel remove` cannot be used with a Solo or Kafka ordering service. “Mixed mode” management, where the system channel is to create channels on some ordering nodes and the new process is used to create channels on other ordering nodes is not supported and highly discouraged. You must either transition to the new process or continue to use the system channel process. For information about removing the system channel from an existing ordering service as part of transitioning to the new process, check out [Remove the system channel](#remove-the-system-channel).

Reviewer: explain here the need to move from system channel to channel participation API, in v2.5, prior upgrading to v3
